src/passes/OptimizeInstructions.cpp

```diff
@@ -121,7 +121,9 @@ struct LocalScanner : PostWalker<LocalScanner> {
   Index getMaxBitsForLocal(LocalGet* get) { return getBitsForType(get->type); }
 
   Index getBitsForType(Type type) {
-    TODO_SINGLE_COMPOUND(type);
+    if (!type.isBasic()) {
+      return -1;
```

Comment:
It's safer this way, as "0" means no bits are used, which might lead an optimization to think that it can do certain things to it. But it would be a bug either way that we should fix, so it doesn't matter much - this just keeps using "-1" as the existing code lower down does.